lms/static/styles/components/_FileList.scss

```diff
@@ -1,58 +1,16 @@
 @use "../variables" as var;
 
 .FileList {
-  position: relative;
-
-  display: flex;
-  flex-direction: column;
-
-  // Override the default minimum height of the table so that the file list
-  // is fully visible even if available height is very small.
-  min-height: 130px;
-}
-
-.FileList__name-header {
-  text-align: left;
-  padding-left: 10px;
-}
-
-.FileList__date-header {
-  width: 120px;
-  text-align: left;
-}
-
-.FileList__icon {
-  // TODO: Update when updated SVGIcon component is available and has sizing
-  // patterns
-  width: 1.5em;
-  height: 1.5em;
-}
-
-.FileList__name-col {
-  overflow: hidden;
-  text-overflow: ellipsis;
-  white-space: nowrap;
-}
-
-.FileList__name {
-  flex-grow: 1;
-  text-decoration: none;
-  color: inherit;
-  vertical-align: middle;
-}
-
-.FileList__date-col {
-  padding-right: 10px;
-}
-
-.FileList__no-files-message {
-  display: flex;
-  justify-content: center;
-  flex-direction: column;
-  vertical-align: middle;
-  text-align: center;
-  width: 100%;
-  min-height: 149px;
-  border: 1px solid var.$grey-6;
-  background-color: var.$grey-2;
+  // FIXME: Reduce specificity when possible to do so
+  // See https://github.com/hypothesis/frontend-shared/issues/180
+  & &__date-header {
```

Comment:
As linked, you see, I'll track this down and fix it in `frontend-shared` next week. Always learning...